lib/datadog_logger.dart

```diff
@@ -88,11 +91,13 @@ class DatadogLogger {
   }
 
   String _levelAsStatus(Level level) {
-    if (level.value <= 500) return 'debug';
+    // Capture 'fine' logs as well as `SHOUT` since SHOUT
+    // is intended for debugging
+    if (level.value <= 500 || level == Level.SHOUT) return 'debug';
```

Comment:
SHOUT logs are intended for debugging? I know the doc reads `Key for extra debugging loudness ([value] = 1200).` but it seems....odd that the log levels would go from error to back to start

Reply:
That's what the docs lead me to believe, but I think the true decision would be up to you -- would you prefer to keep treating these as `critical`?

Reply:
I think ultimately you're right...so yeah don't change it. How unusual.